src/toil_scripts/batch_alignment/bwa_alignment.py

```diff
@@ -174,16 +175,24 @@ def return_input_paths(job, work_dir, ids, *args):
 
     return paths.values()
 
+def mock_mode():
```

Comment:
Duplication of code within a single source tree is nuts. We worked so hard to get rid of the dupes and this is sneaking it back in. Also, proximity isn't the same simplicity. 

Putting it in an `__init__.py` should prevent the dependency issue you mention.

Feel free to discuss in person.